src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue22104.cs

```diff
@@ -15,6 +15,7 @@ public Issue22104(TestDevice testDevice) : base(testDevice)
 		[Test]
 		[Category(UITestCategories.CollectionView)]
 		[FailsOnMac]
+		[FailsOnIOS]
```

Comment:
Could you include a comment here indicating is failing because a bug? Maybe something like:

`[FailsOnIOS("A short description - link to the issue if exists")]`